src/installer/installer.go

```diff
@@ -133,6 +133,11 @@ func (i *installer) InstallNode() error {
 		}
 		i.log.Info("Setting bootstrap node new role to master")
 
+	} else if i.Config.Role == string(models.HostRoleWorker) {
```

Comment:
i am not sure 100% but there is an option it will break day2 flow, as there will be none masters in the cluster